app/routes/reminders/new.js

```diff
@@ -0,0 +1,5 @@
+//ROUTES
```

Comment:
Technically you dont need this file if you're not returning anything in the export function.